java/google/registry/flows/custom/DomainCreateFlowCustomLogic.java

```diff
@@ -92,6 +95,17 @@ public BeforeResponseReturnData beforeResponse(BeforeResponseParameters paramete
      */
     public abstract int years();
 
+    /**
+     * The ID of the validated signed mark.
+     *
+     * <p>If a signed mark was not supplied this value will be null</p>
+     */
+    @Nullable
+    public abstract String signedMarkId();
+
+    /** The time at which this flow was executed */
+    public abstract DateTime transactionTime();
```

Comment:
Oh ok, I saw that being used to get the `DateTime now` in DomainCreateFlow but didn't realize ofy was keeping track of each transaction. I'll update this.